ts/features/idpay/initiative/details/screens/OperationsListScreen.tsx

```diff
@@ -0,0 +1,140 @@
+    importantForAccessibility={"no-hide-descendants"}
+    testID={"activityIndicator"}
+  />
+);
+const renderItem = ({ item }: { item: OperationListDTO }) =>
+  TimelineOperationListItem({ operation: item });
+
+const keyExtractor = (operation: OperationListDTO) => operation.operationId;
+const toast = () => showToast("Errore nel caricamento, riprova.");
+
+export const OperationsListScreen = () => {
+  const route = useRoute<OperationsListScreenRouteProps>();
+  const { initiativeId } = route.params;
+
+  const [isRefreshing, setIsRefreshing] = React.useState(false);
+  const isFirstRenderDispatchedRef = React.useRef(false);
+  const { isLoading, timeline, fetchNextPage, resetTimeline, currentPage } =
+    useInitiativeTimelineFetcher(initiativeId, 10, toast);
+
+  const shouldRenderLoader = isLoading && !isRefreshing;
+  const isFirstLoading = isFirstRenderDispatchedRef.current
+    ? timeline.operationsRecord.length === 0 && shouldRenderLoader
+    : true;
+
+  const refreshTimeline = () => {
+    if (isRefreshing) {
+      return;
+    }
+    setIsRefreshing(true);
+    resetTimeline();
+  };
+  const renderLoader = () => (shouldRenderLoader ? loader() : null);
+
+  useOnFirstRender(() => {
+    resetTimeline();
+    // eslint-disable-next-line functional/immutable-data
+    isFirstRenderDispatchedRef.current = true;
+  });
+  React.useEffect(() => {
+    if (currentPage >= 0 && isRefreshing) {
+      setIsRefreshing(false);
+    }
+  }, [currentPage, isRefreshing]);
+  const pageContent = (
+    <SafeAreaView>
+      <View style={IOStyles.horizontalContentPadding}>
+        <H1>
+          {I18n.t(
+            "idpay.initiative.details.initiativeDetailsScreen.configured.operationsList.title"
+          )}
+        </H1>
+        {isRefreshing ? null : (
+          <Body>
+            {I18n.t(
+              "idpay.initiative.details.initiativeDetailsScreen.configured.operationsList.lastUpdated"
+            )}
+            <Body weight="SemiBold">
+              {formatDateAsLocal(timeline.lastUpdate, true)}
+            </Body>
+          </Body>
+        )}
+      </View>
+      <NBView spacer large />
+      <FlatList
+        style={IOStyles.horizontalContentPadding}
+        contentContainerStyle={styles.listContainer}
+        data={timeline.operationsRecord}
+        keyExtractor={keyExtractor}
+        renderItem={renderItem}
+        onEndReached={fetchNextPage}
+        onEndReachedThreshold={0.5}
+        onRefresh={refreshTimeline}
+        refreshing={isRefreshing}
+        ListFooterComponent={renderLoader}
+      />
+    </SafeAreaView>
+  );
+
+  return (
+    <BaseScreenComponent
+      headerTitle={I18n.t(
+        "idpay.initiative.details.initiativeDetailsScreen.configured.operationsList.header"
+      )}
+      goBack={true}
+    >
+      <LoadingSpinnerOverlay isLoading={isFirstLoading} loadingOpacity={100}>
+        {isFirstLoading ? null : pageContent}
```

Comment:
Why? Shouldn't `LoadingSpinnerOverlay` already hide all the page content if `isFirstLoading ` is `true`?